ccloud/beginner-cloud/start.sh

```diff
@@ -295,55 +287,20 @@ echo "../../ccloud/ccloud-generate-cp-configs.sh $CLIENT_CONFIG &>/dev/null"
 ../../ccloud/ccloud-generate-cp-configs.sh $CLIENT_CONFIG &>/dev/null
 echo "source delta_configs/env.delta"
 source delta_configs/env.delta
+sed -i '' 's/ESN5FSNDHOFFSUEV/$CLOUD_KEY/g' datagen_ccloud_pageviews.json
```

Comment:
@awalther28 clever solution 🎉 !  Now the file renders in the docs with the sample credentials, but also works for automation.

cc: @rspurgeon for review, especially if we want to evaluate this approach for other examples as well

Reply:
One thing I felt was hacky with this route was that sed changes the config file so I added a `git checkout datagen_ccloud_pageviews.json` after the connector is up and running. This way if we are testing changes with start.sh, we won't accidentally add the changed config file. Open to other ideas on the rendering/injecting creds/cleanup/etc.

Reply:
I see what you mean.  The alternative is to leave the file with `$CLOUD_KEY` etc params, and tell the user to substitute their own credentials (which is the route other examples use).

Reply:
@ybyzek Would the preferred route here be to leave the `$CLOUD_KEY`? Just to note, the user would need to manually set the env params, this PR suggests removing the following steps that used to set the env params. If we would prefer users leverage the generate-cp-configs I can add that step back. 
```
../../ccloud/ccloud-generate-cp-configs.sh /tmp/client.config &>/dev/null
source delta_configs/env.delta
```
Although I think manually substituting the key + secrets is more inline with other steps in this tutorial (see "Create a local configuration file /tmp/client.config..." [step 5](https://docs.confluent.io/platform/current/tutorials/examples/ccloud/docs/beginner-cloud.html#create-a-new-service-account-with-an-api-key-secret-pair)).

Reply:
Another option to throw into the mix 😄 : to avoid the hacky `git checkout datagen_ccloud_pageviews.json`, the `start.sh` script can work off a _copy_ of the json connector file and do a substitution to use the temporary credentials.

Reply:
So many options, oh my! @rspurgeon do you have a preference here? 

A quick recap, we're trying to decide the best route to present a fully managed connector config file and how to use that config file in an automated stat script. 
The options appear to be: use env vars and add the process to set them in the tutorial instructions, create a unique config file for the start script and the tutorial rendering, or use the same config file for both and for the automation leverage `sed` to input secrets +`git checkout` to revert the config file. 

As stated earlier, I think the latter two better fit with the tutorial.

Reply:
@ybyzek I spoke with @rspurgeon offline. The conclusion we came to is to have the `start.sh` script work off of a copy of the json connector file housed in a directory that is ignored by bit. This is something similar to how kafka-devops works. The work has been validated by myself and pushed up to this branch.
 To my knowledge, this is the last standing issue for this PR.

Reply:
@awalther28 Thanks for working through that, the approach looks good.